csrankings-p.csv

```diff
@@ -534,7 +534,7 @@ Pengyu Hong,Brandeis University,http://www.cs.brandeis.edu/~hong,pvDa8pcAAAAJ
 Penny Kyburz,Australian National University,https://cs.anu.edu.au/people/penny-kyburz,mrvBuvsAAAAJ
 Penny Rheingans,Univ. of Maryland - Baltimore County,http://www.csee.umbc.edu/~rheingan,1k0rojcAAAAJ
 Penny Sanderson,University of Queensland,https://www.psy.uq.edu.au/people/personal.html?id=533,eMywJ4sAAAAJ
-Penny Sweetser,Australian National University,https://cs.anu.edu.au/people/penny-kyburz,mrvBuvsAAAAJ
+Penny Kyburz,Australian National University,https://cs.anu.edu.au/people/penny-kyburz,mrvBuvsAAAAJ

```

Comment:
Needs entries for all aliases (Penny Sweetser and Penelope Sweetser).

Reply:
Removed duplicate entry. DBLP now has her as just Penny Kyburz.